pkg/util/env.go

```diff
@@ -129,6 +153,20 @@ func getEnvReplicaCount(envVarName string, defaultValue int32) int32 {
 	return defaultValue
 }
 
+// getBoolean returns the boolean value for an env var; if not set, false is returned
+func getBoolean(varName string) bool {
+	svalue, ok := LookupEnvFunc(varName)
```

Comment:
I agree with Paul on making these enabled by default. It looks like they would all be disabled by default per this logic

Reply:
OK.  The charts handle it, but if we want to enforce here we can.

Reply:
fixed